### PR TITLE
fixes for imod protocols

### PR DESCRIPTION
**Pull request by azazellochg**
Some changes for default params in imod protocols so that they are more adapted for cryo-em:

- always use antialias when binning a stack
- two surfaces search is off in both protocols for consistency
- bead radius default is 18px (good range for bin 4 and 10nm beads)
- use sobel filter for bead centering for most cryo data
- use lowpass for bead tracking
- use adjustbeadsizes
- trimvol -yz replaced by -rx, the first one flips the hand. I'm happy to dicuss this as it's not clear to me why -yz is used now
- add expandcircles to eraser
- replace wrong dose file format for mtffilter
- set output 3D coord size based on bead diameter
- fix betterradius param in gold eraser tab of the fid alignment
- add missing pix size in fid align
- fix minpacing in gold picker
- add missing taper arg for newstack

Other issues to discuss: 

- how about replacing excludeviews by newstack? The advantage of using excludeviews is being reversible, but I dont see the point of doing this in scipion. Original stacks are kept anyway.
- how about adding gpu/parallel options to etomo protocol?
- report rmsd and the ratio in the summary of fiducial alignment
- report number of points in each landmark model in the viewer, report number of gaps in each model
- add a possibility to display imod's log files from each operation, I think we have a textviewer for this

The tests are passing and I will go through hiv data with these changes to double check

### CODE REVIEW

File: imod/protocols/protocol_applyTransformationMatrix.py [hunk 1]
```diff
@@ -97,6 +97,7 @@ def computeAlignmentStep(self, tsObjId):
                         "-output %(output)s " \
                         "-xform %(xform)s " \
                         "-bin %(bin)d " \
+                        "-antialias -1 " \
```
Comment (pconesa) [inline imod/protocols/protocol_applyTransformationMatrix.py:100]: I have a local change for this with a parameter, so user can choose the method.

**Comment by azazellochg** [comment]
Hi, I don't mind having a patam, just make sure the default is 6 (-1).
Also, this param will need to be added to many protocols

File: imod/protocols/protocol_xCorrPrealignment.py [hunk 2]
```diff
@@ -70,7 +70,7 @@ def _defineParams(self, form):
         form.addParam('computeAlignment',
                       params.EnumParam,
                       choices=['Yes', 'No'],
-                      default=1,
```
[the remark this answers is not in the document]
Comment (azazellochg) [reply imod/protocols/protocol_xCorrPrealignment.py]: ok! done!

File: imod/protocols/protocol_applyTransformationMatrix.py (hunk 1 above)
Comment (pconesa) [reply imod/protocols/protocol_applyTransformationMatrix.py:100]: Let's merge this first and then my branch.

Comment (azazellochg) [reply imod/protocols/protocol_applyTransformationMatrix.py:100]: I'm still fixing things though...

**Comment by azazellochg** [comment]
@pconesa @fede-pe this is now ready for review. I won't make more fixes here, they can be addressed later

**Review by pconesa — APPROVED** [review]
Thanks @azazellochg. I do not have any objection. All if fine for me.

**Pull request merged**